portfolio/src/main/webapp/style.css

```diff
@@ -4,6 +4,15 @@
   width: 650px;
 }
 
-#greeting-container {
+#fact-container {
```

Comment:
Is this still in use?

Reply:
This was part of the tutorial for Week 2. It is in use for that starter website but not for my Angular version.